src/endpoint/iam/iam_utils.js

```diff
@@ -82,15 +82,47 @@ function check_iam_path_was_set(iam_path) {
     return iam_path && iam_path !== IAM_DEFAULT_PATH;
 }
 
+/**
+ * parse_max_items converts the input to the needed type
+ * assuming that we've got only sting type
+ * @param {any} input_max_items
+ */
+function parse_max_items(input_max_items) {
+    if (input_max_items === undefined) return;
+    const input_type = 'number';
+    const parameter_name = 'MaxItems';
```

Comment:
these consts seem to be unneeded

Reply:
I'm using them in the error message (and prefer a template string).